datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Domain/SidebarDomainSection.tsx

```diff
@@ -45,7 +45,7 @@ export const SidebarDomainSection = () => {
     };
 
     return (
-        <div>
+        <div id="entity-profile-domains">
```

Comment:
I think it'd be great to make all of these IDs global constants to be shared in `sources.tsx`, in the `id`s here, as well as the `stepIds` that we pass into `OnboardingTour` component

Reply:
Refactored!